test/utils/runners.go

```diff
@@ -1537,6 +1541,13 @@ func attachConfigMaps(template *v1.PodTemplateSpec, configMapNames []string) {
 	template.Spec.Containers[0].VolumeMounts = mounts
 }
 
+func (config *RCConfig) getTerminationGracePeriodSeconds(defaultGrace *int64) *int64 {
```

Comment:
Why not just have the application logic at the use sites apply whatever logic they want? Do we have similar "get default value in certain situations" methods for all the other values?

In what situations does the default change? Why not use the same default throughout and make it a constant?

I'm not against having a method like this, but it seems slightly unnecessary and maybe even a bit confusing. If I was creating an object, I wouldn't know what the default should be and would expect that to just be set if there was a helper method like this. Otherwise this method is just a specific use case of

```
func firstPositiveInt64Val(vals ...*int64) *int64{
   for _,v := range vals{
    if v!=nil && *v>0 {return v}
  }
}
```

Reply:
It seems this has maybe already been talked about by other comments and is otherwise lgtm/approved. Disregard if everyone is happy with it, but had just gotten around to review.